api/edge_api/identities/views.py

```diff
@@ -94,8 +103,34 @@ def get_environment_from_request(self):
     def perform_destroy(self, instance):
         Identity.dynamo_wrapper.delete_item(instance["composite_key"])
 
+    @swagger_auto_schema(
+        responses={200: EdgeIdentityTraitsSerializer(many=True)},
+    )
+    @action(detail=True, methods=["get"], url_path="list-traits")
+    def get_traits(self, request, *args, **kwargs):
+        identity = self.get_object()
+        data = trait_schema.dump(identity["identity_traits"], many=True)
+        return Response(data=data, status=status.HTTP_200_OK)
+
+    @swagger_auto_schema(
+        method="put",
+        request_body=EdgeIdentityTraitsSerializer,
+        responses={200: EdgeIdentityTraitsSerializer},
+    )
+    @action(detail=True, methods=["put"], url_path="update-traits")
+    def update_traits(self, request, *args, **kwargs):
+        identity = build_identity_model(self.get_object())
+        try:
+            trait = trait_schema.load(request.data)
+        except marshmallow.ValidationError as validation_error:
+            raise ValidationError(validation_error) from validation_error
+        identity.update_traits([trait])
+        Identity.dynamo_wrapper.put_item(build_identity_dict(identity))
```

Comment:
We need to make sure that the organisation is configured to persist traits.

Reply:
Done